packages/nft-exchange-transactions/src/handlers/nft-exchange-handler.ts

```diff
@@ -1,8 +1,28 @@
 import { Handlers } from "@arkecosystem/core-transactions";
-import { Managers } from "@arkecosystem/crypto";
+import { Managers, Utils } from "@arkecosystem/crypto";
+import { Contracts } from "@packages/core-kernel";
+
+import { defaults } from "../defaults";
+import { FeeType } from "../enums";
 
 export abstract class NFTExchangeTransactionHandler extends Handlers.TransactionHandler {
     public async isActivated(): Promise<boolean> {
         return Managers.configManager.getMilestone().aip11 === true;
     }
+
+    public dynamicFee({
+        addonBytes,
+        satoshiPerByte,
+        transaction,
+        height,
+    }: Contracts.Shared.DynamicFeeContext): Utils.BigNumber {
+        if (defaults.feeType === FeeType.Static) {
```

Comment:
Use defaults from injected configuration instead of using it directly from default file.